dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,67 @@
+{% macro  bigquery__get_show_grant_sql(relation) %}
+    select privilege_type, grantee from {{ relation.project }}.`region-{{ target.location }}`.INFORMATION_SCHEMA.OBJECT_PRIVILEGES
+    where object_schema = "{{ relation.dataset }}" and object_name = "{{ relation.identifier }}"
```

Comment:
Fair.

Is there a less yucky way than `and grantee != "user:"||session_user()`?  Just `session_user()` won't work since the grantee has `user:` prepended to it.

Reply:
Of course it's not all `users`...